src/inmanta/parser/plyInmantaParser.py

```diff
@@ -1027,9 +1035,9 @@ def convert_to_references(variables: List[Tuple[str, LocatableString]]) -> List[
                 ref.location = range_attr
                 ref.namespace = namespace
             # For a composite variable e.g. 'a.b.c', we only add the reference to the innermost attribute (e.g. 'c')
-            _vars.append((ref, match))
+            _vars.append((ref, " " * padding[0] + match + " " * padding[1]))
         else:
-            _vars.append((ref, match))
+            _vars.append((ref, " " * padding[0] + match + " " * padding[1]))
```

Comment:
Why add the padding back?

What if the original padding is not ' '  but e.g. '\t' ?